### PR TITLE
Edits to the documentation

### PR DESCRIPTION
**Pull request by PadowYT2**
## Changes in this PR
- Update supported PHP ranges to 8.1 - 8.3
- Updated recommended/default PHP version to 8.3
- Replaced (unsupported) CentOS with RHEL / Rocky Linux / AlmaLinux
- Added some missing comments to web server configs
- Updated footer copyright text
- Swaps back to nodesources install scripts as these are now supported again
- Adds Ubuntu 24.04 to the list of supported OSes
- Minor formatting changes in web server configs
- Minor wording and formatting changes

### CODE REVIEW

File: .vuepress/theme/Home.vue [hunk 1]
```diff
@@ -146,7 +146,7 @@
                         </a>
                     </div>
                 </div>
-                <div class="footer">MIT Licensed | Pterodactyl&reg; Copyright &copy; 2015 - 2022 Dane Everitt & contributors.</div>
+                <div class="footer">MIT Licensed | Pterodactyl&reg; Copyright &copy; Dane Everitt and contributors</div>
```
Comment (Jelcoo) [inline .vuepress/theme/Home.vue:149]: This is also not exclusive copyright to Dane anymore

File: .snippets/webservers/nginx-php8.1-nossl.conf [hunk 2]
```diff
@@ -1,9 +1,7 @@
 server {
-    # Replace the example <domain> with your domain name or IP address
```
Comment (Boy132) [inline .snippets/webservers/nginx-php8.1-nossl.conf:2]: This shouldn't be removed IMO. Yes, it's a bit redundant but _many_ users easily overlook `<domain>` so there is no harm in leaving it.

File: .vuepress/theme/Home.vue (hunk 1 above)
Comment (PadowYT2) [reply .vuepress/theme/Home.vue:149]: Well, pterodactyl/panel (and maybe) pterodactyl/wings still has Dane, so maybe we should keep it...?

File: .snippets/webservers/nginx-php8.1-nossl.conf (hunk 2 above)
Comment (PadowYT2) [reply .snippets/webservers/nginx-php8.1-nossl.conf:2]: Just looks weird without having the text in configs older than PHP 8.1, bu alright

Comment (PadowYT2) [reply .snippets/webservers/nginx-php8.1-nossl.conf:2]: although, should be this added into SSL configs aswell as Apache? I still don't see a point in this comment

**Comment by rowansc1** [comment]
@PadowYT2 - Is this still something you want to merge eventually? Although it's been like 8 months lol

**Comment by PadowYT2** [comment]
> @PadowYT2 - Is this still something you want to merge eventually? Although it's been like 8 months lol

Sorry, but still waiting on the opinion on the original comment

File: .snippets/webservers/nginx-php8.1-nossl.conf (hunk 2 above)
Comment (danny6167) [reply .snippets/webservers/nginx-php8.1-nossl.conf:2]: Yes this needs to be kept as well as added to any configs that it's missing from

File: .vuepress/theme/Home.vue (hunk 1 above)
Comment (danny6167) [reply .vuepress/theme/Home.vue:149]: There was a commit recently by Matthew where he touched the license / copyright line in the about project page.
I would infer his preference from that.
https://github.com/pterodactyl/documentation/commit/cc2770a8a7661a9a1611338a003520be1eec7db4

**Comment by danny6167** [comment]
For those playing along at home, I've been pushing builds of this to CF pages as I go along.
https://pr-553.ptero-doc-preview.pages.dev/

**Comment by PadowYT2** [comment]
Didn't realize you are pushing your changes to this PR, I guess thanks. Also, why PHP 8.3 and not 8.2?

**Comment by danny6167** [comment]
We wanted to get this PR finished and merged, but it touched so many different files we didn't want to work on other PRs for the other stuff and risk having to deal with the merge conflicts afterwards. We're happy to close this and open a different PR if you would prefer. 

PHP 8.3 is fully released and compatible with Pterodactyl. I didn't see any reason to target 8.2 when it's only in active support for another 5 months.

**Comment by rowansc1** [comment]
I'm happy with this in its current state 👍 well done

File: panel/1.0/getting_started.md [hunk 3]
```diff
@@ -23,18 +23,19 @@ Pterodactyl does not support most OpenVZ systems due to incompatibilities with D
...
+|                                    | 24.04   | :white_check_mark: | MariaDB can be installed without the repo setup script.     |
+| **RHEL / Rocky Linux / AlmaLinux** | 8       | :white_check_mark: | Extra repos are required.                                   |
+|                                    | 9       | :white_check_mark: |                                                             |
+| **Debian**                         | 11      | :white_check_mark: |                                                             |
+|                                    | 12      | :white_check_mark: |                                                             |
 
 ## Dependencies
 
-* PHP `8.0` or `8.1` (recommended) with the following extensions: `cli`, `openssl`, `gd`, `mysql`, `PDO`, `mbstring`, `tokenizer`, `bcmath`, `xml` or `dom`, `curl`, `zip`, and `fpm` if you are planning to use NGINX.
+* PHP `8.1`, `8.2`, or `8.3` (recommended) with the following extensions: `cli`, `openssl`, `gd`, `mysql`, `PDO`, `mbstring`, `tokenizer`, `bcmath`, `xml` or `dom`, `curl`, `zip`, and `fpm` if you are planning to use NGINX.
```
Comment (rowansc1) [inline panel/1.0/getting_started.md:38]: We should consider that php 8.1 is in security-only support, but that will stop in Dec 2025 so there is plenty of time. We just need to remember to remove it from the Docs when that date comes close.

**Review by rowansc1 — COMMENTED** [review]
All happy with this! Just need Matt to approve and merge :)

File: panel/1.0/getting_started.md (hunk 3 above)
Comment (PadowYT2) [reply panel/1.0/getting_started.md:38]: should be easily doable. 8.5 would be out by that time

**Review by PadowYT2 — COMMENTED** [review]
If PHP 8.3 is used by default, we don't need `nginx-php8.1-*.conf`

**Review by rowansc1 — COMMENTED** [review]
Sweet. I like the small changes here. I don't think the table changes were strictly needed, but definitely makes the markdown easier to read.

Should be good to go when matt reviews it. Thanks for coming back to it after so long @PadowYT2!

**Review by matthewpi — APPROVED** [review]

**Pull request merged**